UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -87,6 +90,9 @@ public IPlayerPossessable PlayerPossessable
 	private ObservableCollection<Spell> spells = new ObservableCollection<Spell>();
 	public ObservableCollection<Spell> Spells => spells;
 
+	private readonly ObservableCollection<ChangelingAbility> changelingAbilities = new ObservableCollection<ChangelingAbility>();
```

Comment:
Strange question... But it can be merge but it may create problems with code readability and need anyway something for splitting spells from abilities... So don`t know it's really a good idea

Reply:
I don't think its good to hardcode changelings into base systems. You should restructure this to be much more modular as per our [development standards](https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#1-writing-modular-code-with-minimal-dependencies)

Reply:
Ok, I move this from mind to changeling

Reply:
please don't resolve conversations on your own. I still see changeling stuff here so this clearly isn't resolved.

Reply:
Ok, just marking that this thing is resolved on my local git, gonna upload that later